V1.0.0 RC (#182)
OWASP/maswe: weaknesses/MASVS-PLATFORM/MASWE-0037.md

Author: Carlos: DONE

---
title: WebViews Allow Access to Local Resources with Untrusted Content
id: MASWE-0037
alias: webviews-local-resources

> requirement: "The app does not allow WebViews loading untrusted content to access local resources."

Reviewer: ```suggestion
requirement: "The app only allows trusted WebView content to access local resources."
```
Suggested rephrasing to make it more consistent.
